From 065cf8419cdf23b24ef85f5a936f6962f4403d3a Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Wed, 8 Nov 2023 16:10:13 +0000
Subject: [PATCH 399/697] dt-bindings: mmc: sdhci-of-dwcmhsc: Add Raspberry Pi
 RP1 support

The DWC MSHC controller on RP1 needs differentiating from the generic
version.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 .../devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml          | 5 +++++
 1 file changed, 5 insertions(+)

--- a/Documentation/devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml
+++ b/Documentation/devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml
@@ -17,6 +17,7 @@ properties:
           - const: rockchip,rk3576-dwcmshc
           - const: rockchip,rk3588-dwcmshc
       - enum:
+          - raspberrypi,rp1-dwcmshc
           - rockchip,rk3568-dwcmshc
           - rockchip,rk3588-dwcmshc
           - snps,dwcmshc-sdhci
@@ -95,6 +96,8 @@ allOf:
             - description: axi clock for rockchip specified
             - description: block clock for rockchip specified
             - description: timer clock for rockchip specified
+            - description: timeout clock for rp1 specified
+            - description: sdio clock generator for rp1 specified
         clock-names:
           minItems: 1
           items:
@@ -103,6 +106,8 @@ allOf:
             - const: axi
             - const: block
             - const: timer
+            - const: timeout
+            - const: sdio
 
   - if:
       properties:
